hieradata/common.yaml

```diff
@@ -712,6 +712,7 @@ govuk_ci::master::pipeline_jobs:
   govuk_security_audit: {}
   govuk_sidekiq: {}
   govuk_template: {}
+  govuk_taxonomy_helpers: {}
```

Comment:
Should be before `govuk_template` to keep the list in order, but otherwise this is correct. 👍